### DISCUSSION TITLE
Filter on content or sections

### DISCUSSION THREAD

**Discussion by akib1689** [rendercv/rendercv#163] [Q&A]
I don't know if I'm missing something or not. It would be great if we would hide/include some of the sections from the `yaml` file. 

**My Use case:**

I apply for 2/3 roles. and each time I need to manually comment/un-comment the relevant portion of the yaml file. 
It would be very useful if we had something like tag in each item. 
> [!NOTE]
> Each section will have some tags. say one of my skill row should be on devops related jobs. and this row should not be in developer related jobs. It would be really helpful to set match tag from command line say devops. and each node that has the tag devops. will be in that particular render of CV. by default it will take all the contents. 

The Implementation can be flexible. In documentation I could not find anything related to filtering content of the CV. That is why I'm asking is here.

**Comment by sinaatalay** [comment]
Thank you for the idea. Could you please open an issue with an enhancement tag for this?
